packages/devtools/lib/src/inspector/inspector_tree.dart

```diff
@@ -324,7 +332,7 @@ abstract class InspectorTreeNode {
 
   void dirty() {
```

Comment:
make this private or move it into the isDirty setter.

Reply:
done.